src/fides/api/ops/service/connectors/email_connector.py

```diff
@@ -28,202 +24,104 @@
     MessagingServiceType,
 )
 from fides.api.ops.schemas.redis_cache import Identity
-from fides.api.ops.service.connectors.base_connector import BaseConnector
-from fides.api.ops.service.connectors.query_config import ManualQueryConfig
+from fides.api.ops.service.connectors.base_connector import LimitedConnector
 from fides.api.ops.service.messaging.message_dispatch_service import dispatch_message
-from fides.api.ops.util.collection_util import Row, append
-from fides.lib.models.audit_log import AuditLog, AuditLogAction
-
 
-class EmailConnector(BaseConnector[None]):
```

Comment:
Removing dataset logic from `EmailConnector` we will only pass identities for erasure emails moving forward.